src/material/button/button.scss

```diff
@@ -153,6 +153,22 @@
   }
 }
 
+// By default, all focus indicators are flush with the bounding box of their
+// host element. For the button element, default inset/offset
+// values are necessary to ensure that the focus indicator is sufficiently
+// contrastive and renders appropriately.
+.mat-focus-indicator.mat-flat-button::before,
+.mat-focus-indicator.mat-raised-button::before,
+.mat-focus-indicator.mat-fab::before,
+.mat-focus-indicator.mat-mini-fab::before {
+  margin: calc(-(var(--mat-focus-indicator-border-width, $default-focus-indicator-width) + 2px));
+}
+
+.mat-focus-indicator.mat-stroked-button::before
+{
```

Comment:
This formatting doesn't look quite right, I'd expect this `{` to be on the same line above.